docker-compose.elasticsearch.yaml

```diff
@@ -21,8 +21,10 @@ services:
     volumes:
       - elasticsearch:/usr/share/elasticsearch/data
       - ".:/mnt/ddev_config"
+      # Uncomment the following line if using elasticsearch 8
+      # - ./elasticsearch/config/elasticsearch-8.yml:/usr/share/elasticsearch/config/elasticsearch.yml
     healthcheck:
-      test: ["CMD-SHELL", "curl --fail -s localhost:9200"]
+      test: ["CMD-SHELL", "curl --fail -s elasticsearch:9200"]
```

Comment:
Override this in another file

Reply:
This is a bug in the core yaml and should be changed. The `-s` hid the output all the time: `curl: (7) Failed to connect to localhost port 9200: Connection refused`

Now it is the same as in the current test file: https://github.com/ddev/ddev-elasticsearch/blob/main/tests/test.bats#L28